variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/AbstractVariantSourceEntry.java

```diff
@@ -65,7 +66,10 @@ public AbstractVariantSourceEntry(String fileId, String studyId, String[] second
         this.fileId = fileId;
         this.studyId = studyId;
         if (secondaryAlternates != null) {
-            this.secondaryAlternates = Arrays.copyOf(secondaryAlternates, secondaryAlternates.length);
+            this.secondaryAlternates = Arrays.stream(secondaryAlternates)
```

Comment:
we should call setSecondaryAlternates here...

Reply:
done